debian/rules

```diff
@@ -136,5 +156,8 @@ override_dh_auto_install:
 override_dh_installinit:
 	dh_installinit -pcloudstack-management -pcloudstack-agent -pcloudstack-usage --onlyscripts --no-start
 
+override_dh_systemd_enable:
+	dh_systemd_enable -pcloudstack-agent -pcloudstack-usage --with systemd
```

Comment:
I didn't mean the `--with systemd` to be here, just on the `dh` invocation in the `%` target (like it now is).